README.md

```diff
@@ -129,7 +129,7 @@ const {
   reducer,
   initial,
   slice
-} = counter;
+} = counter();
```

Comment:
`counter` is an object created by the `autodux()` call. This would introduce an error.